src/core/helpers.py

```
@@ -14,29 +14,13 @@
 # limitations under the License.
 #
 
-"""Low-level utilities – date helpers, hashing,
-path normalisation, and subprocess wrappers for the ``gh`` CLI.
-"""
+"""Pure utility functions – date helpers, hashing, and path normalisation."""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)